source/tests/PuppTests.test.js

```diff
@@ -54,7 +54,7 @@ describe('basic navigation for BJ', () => {
         await page.waitForTimeout(1000);
     });
 
-    it('Test1: Initial Home Page - Shows create your login ', async () => {
```

Comment:
I tried running the Puppeteer tests with your code and it failed a lot so maybe you can take time to figure out what is going on or just send a screenshot that it works on your machine
![image](https://user-images.githubusercontent.com/76570554/171767834-bce60cfe-8454-40ae-86e3-d4b24be1efc3.png)